conda_smithy/ci_register.py

```diff
@@ -188,8 +188,9 @@ def add_project_to_travis(user, project):
 
         response = requests.get(url, headers=headers)
         try:
+            response.raise_for_status()
             content = response.json()
-        except ValueError:
+        except Exception:
```

Comment:
Couldn't we do better than a generic exception here? Looks like this should be an [`HTTPError`]( http://docs.python-requests.org/en/master/api/#requests.Response.raise_for_status ) for the status. I don't know if the `ValueError` from trying to parse the JSON still needs to be handled, but I guess we could do that too if needed.